components/twitter/actions/delete-tweet/delete-tweet.mjs

```diff
@@ -0,0 +1,37 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-delete-tweet",
+  name: "Delete Tweet",
+  description: "Remove a posted tweet",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    twitter,
+    tweetID: {
+      propDefinition: [
+        twitter,
+        "tweetID",
+      ],
+    },
+    trimUser: {
+      propDefinition: [
+        twitter,
+        "trimUser",
+      ],
+    },
+  },
+  async run() {
+    const {
+      tweetID,
+      trimUser,
+    } = this;
+
+    const params = {
+      tweetID,
+      trimUser,
+    };
+
+    return await this.twitter.deleteTweet(params);
```

Comment:
Since this is an async function. [You don't need to `return await`](https://eslint.org/docs/rules/no-return-await). Could you please remove it?